core/src/main/resources/application.conf

```diff
@@ -84,6 +84,10 @@ system {
 
   // Since the WorkflowLogCopyRouter is initialized in code, this is the number of workers
   number-of-workflow-log-copy-workers = 10
+
+  // Whether to run the metadata summarizer in this Cromwell instance.  There should only be one metadata summarizer
+  // per database, so if this Cromwell instance is being used in read-only mode this value should be `false`.
```

Comment:
If you think it's a good idea ;)

-------------------------------
Kristian Cibulskis
Chief Architect, Data Sciences & Data Engineering
Broad Institute of MIT and Harvard
kcibul@broadinstitute.org



On Fri, Sep 9, 2016 at 2:15 PM, mcovarr <notifications@github.com> wrote:

> In core/src/main/resources/application.conf
> <https://github.com/broadinstitute/cromwell/pull/1385#discussion_r78225031>
> :
>
> > @@ -84,6 +84,10 @@ system {
> >
> >    // Since the WorkflowLogCopyRouter is initialized in code, this is the number of workers
> >    number-of-workflow-log-copy-workers = 10
> > +
> > +  // Whether to run the metadata summarizer in this Cromwell instance.  There should only be one metadata summarizer
> > +  // per database, so if this Cromwell instance is being used in read-only mode this value should be `false`.
>
> @kcibul <https://github.com/kcibul> sorry, I didn't read the ticket
> thoroughly enough and missed the comment about reusing the refresh
> interval. I'm happy to change this, I think that could actually simplify
> things nicely.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/cromwell/pull/1385/files/37e8a49d1bf7dd628e6035bb9e279bb3d31d922f#r78225031>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABW4g4c50JkC2hcRT1jrQxc5GVvo_4GDks5qoaIogaJpZM4J5PYi>
> .
>